site/docs/uk/hosting/vps.md

```diff
@@ -401,7 +401,7 @@ Deno.serve(async (req) => {
 Далі знайдіть і видаліть запис типу `CNAME` з імʼям `www`.
 Замість нього створіть новий запис типу `A` з імʼям `www`, що вказуватиме на IP-адресу вашого VPS, і встановіть TTL рівним 3600.
 
-> Якщо у вас виникнуть проблеми, скористайтеся іншим способом, описаним в [базі знань](https://support.hostinger.com/uk/articles/1583227-%D1%8F%D0%BA-%D0%BD%D0%B0%D0%BF%D1%80%D0%B0%D0%B2%D0%B8%D1%82%D0%B8-%D0%B4%D0%BE%D0%BC%D0%B5%D0%BD-%D0%BD%D0%B0-vps).
```

Comment:
Is this a change the link-checker suggested, like a redirection?

Reply:
No. I fixed it myself so that all pages have the same link. In the Ukrainian translation, I used the translated knowledge base, while other translators used the English one.

Reply:
I don't know if this is what we do through out the website, but, personally, I would've wanted links that points to websites with the same language I'm reading the documentation in, if possible. Resolving this conversation, now.

Reply:
I think you mean the same thing. Spanish docs should link to Spanish versions of external websites unless they aren't available in Spanish, in which case it should link to English. Same thing for other languages. Disagreements?